internal/prompts/app_select_test.go

```diff
@@ -573,9 +575,8 @@ func Test_FilterAuthsByToken_Flags(t *testing.T) {
 //
 
 func TestPrompt_AppSelectPrompt_SelectedAuthExpired_UserReAuthenticates(t *testing.T) {
-
```

Comment:
vent: Personal pet-peeve is newlines at the top of a function block in Golang 😆

Reply:
➕ I find these make jumping around sections sometimes a hassle- Thanks for reducing our linecounts with this change!